components/elm/src/biogeochem/DUSTMod.F90

```diff
@@ -93,6 +94,8 @@ subroutine Init(this, bounds)
     call this%InitCold     (bounds)
     call this%InitDustVars (bounds)
 
+    write(iulog,*) "elm, DUSTMod.F90, Init: dust_emis_scheme = ",dust_emis_scheme
+
```

Comment:
Same as above for eam's dust_model.F90 during init. Maybe useful to have both as you currently do, for the interest of running I-case.

Reply:
Also made the same change in elm.